### DISCUSSION TITLE
`uosc.conf`配置文件中 --controls 参数可能有错误。

### DISCUSSION THREAD

**Discussion by wathper** [dyphire/mpv-config#61] [General]
```
controls=menu,open-file,<idle>command:history:script-binding recentmenu/open?最近播放,<idle>command:bookmarks:script-binding simplebookmark/open-list?书签菜单,<idle>command:file_copy:script-binding smartcopypaste_II/open-list?剪贴菜单,gap,<!idle>command:analytics:script-binding stats/display-stats-toggle?统计,gap,<stream>command:high_quality:script-binding quality_menu/video_formats_toggle?流媒体品质,<audio,image>command:image:script-binding uosc/video#video?封面,<has_many_edition>editions,<video>video,<has_audio,!audio>audio,<has_sub>subtitles,<has_chapter>chapters,space,<has_chapter>command:skip_previous:add chapter -1?上一章节,<!idle,!image>speed,<has_chapter>command:skip_next:add chapter 1?下一章节,space,loop-file,shuffle,loop-playlist,gap,<has_playlist>prev,<has_playlist>playlist,<has_playlist>next,gap,audio-device,gap,fullscreen
```

该代码块中 `<audio,image>command:image:script-binding uosc/video#video?封面`里的`#video`是多余的吗？

我在 --controls 添加了一些组件并赋予了中文别名。

启动 mpv 后，在` <audio,image>command:image:script-binding uosc/video#video?封面`后面组件的中文名不能被显示，

并且`<audio,image>command:image:script-binding uosc/video#video?封面`后的字符串里的中文也被自动删除。

在将参数`<audio,image>command:image:script-binding uosc/video#video?封面`整个删除后则恢复正常。

**Comment by dyphire** [comment]
这条参数的作用是显示音频文件的封面控件，参数确实有点问题，在提交 https://github.com/dyphire/mpv-config/commit/5d4f8af91538dde29a899f6481c00342cb838525 中修正。但它在语法结构上并没有错误，很可能是你加的自定义参数有问题
参考配置文件内的参数语法的详细解释，`#prop`的解释和用法就在里面
https://github.com/dyphire/mpv-config/blob/ac132e3286950be0b235a72e44e88666e82e44b3/script-opts/uosc.conf#L21-L80

**Reply by wathper** [reply]
那应该就是我之前的参数打错了吧。
